App/localization/languages.old/vi.json

```diff
@@ -0,0 +1,130 @@
+{
+  "components": {
+    "box_footnote": "EMPTY",
+    "box_per_day": "EMPTY",
+    "btn_back": "Back",
+    "cigarette_count": "EMPTY",
+    "cigarette_report": "{{swear_word}}! You smoke\n<3>{{cigarette_count}}</3> {{frequency}}.",
+    "current_location_unknown_station": "EMPTY",
+    "distance_unit": {
+      "long_km": "kilometer",
+      "long_mi": "mile",
+      "short_km": "km",
+      "short_mi": "mi"
+    },
+    "frequency": {
+      "daily": "daily",
+      "monthly": "monthly",
+      "never": "never",
```

Comment:
why are there 2 folders? languages.old and legacy_languages

Reply:
The legacy is script before this PR. The old is new generated script I kept for comparison. The old should be deleted before last PR. My apologize.